src/pages/home-page.js

```diff
@@ -46,41 +53,109 @@ export default function HomePage() {
     setPokemonsDisplay(prePokemons => [...prePokemons, ...pokemonSlice])
   }, [pokemons, offset])
 
+  // Start search when form submit.
+  const handleSubmitSearchForm = e => {
+    e.preventDefault()
+    if (keywords.trim().length > 0) {
+      const stn = 'name'
+      const stt = 'type'
```

Comment:
why do we define these constant in here?

Reply:
i think these constant for look clean code.